static/elements/chromedash-banner.js

```diff
@@ -0,0 +1,59 @@
+import {LitElement, css, html} from 'lit-element';
+import {nothing} from 'lit-html';
+import SHARED_STYLES from '../css/shared.css';
+
+
+class ChromedashBanner extends LitElement {
+  static get properties() {
+    return {
+      timestamp: {type: Number},
+      message: {type: String},
+    };
+  }
+
+  constructor() {
+    super();
+    this.timestamp = (new Date()).getTime() / 1000;
+    this.message = null;
```

Comment:
nit: `this.message = ''` or `nothing`? (to match the property type defined above)

Reply:
Done.